### ISSUE TITLE
Unexpected getNights() result

### ISSUE THREAD

**Issue by cynat** [benitolopez/hotel-datepicker#180]
2024-10-27 selection returns 2 nights instead of 1 (using 4.4.1 release but same result on demo site,
![HDP-Issue-20241027](https://github.com/benitolopez/hotel-datepicker/assets/95433758/6624e40c-3c43-4d48-bc82-e8b3cbfa6bdf)
 see attached).

**Comment by benitolopez** [comment]
Yes, I see. This happens because of the daylight saving time change on the 27th of October.

**Comment by benitolopez** [comment]
Fixed in 4.7.1
